src/lib/cli/exit.ts

```diff
@@ -14,5 +16,9 @@ export function withError( message: Error | string ): never {
 		} ${ env.os.version }`
```

Comment:
Can we use this PR as an opportunity to also add `os.arch()` to the output?

Reply:
Done.